src/pages/accomodation/components/ImagesList.tsx

```diff
@@ -0,0 +1,119 @@
+import { Box, ImageList, ImageListItem, useMediaQuery, useTheme } from '@mui/material';
+import ShowPhotos from '@src/components/full-view-accommodation/full-view-accommodation';
+import Slider from '@src/components/shared/Slider';
+import { AccommodationSingle } from '@src/types/accommodation.types';
+import { useState } from 'react';
+
+export default function ImagesList({ data }: { data: AccommodationSingle }) {
+  const [openDialog, setOpenDialog] = useState(false);
+
+  const handleOpenDialog = () => {
+    setOpenDialog(true);
+  };
+  const handleCloseDialog = () => {
+    setOpenDialog(false);
+  };
+  const [image_1, image_2, image_3, image_4, image_5] = data.media;
```

Comment:
I don't see any reason for destructuring array like this. Why can't you operate it as any other array?